agent/stats/queue.go

```diff
@@ -430,3 +430,50 @@ func (queue *Queue) getULongStatsSet(getUsageInt getUsageIntFunc) (*ecstcs.ULong
 		OverflowSum: &overflowSum,
 	}, nil
 }
+
+// getCWStatsSet gets the stats set for per second network metrics
```

Comment:
Nit: comment does not match the name of the function. It would be helpful to explain what you mean by UDouble here too? Is it unsigned double type?

Reply:
Thanks for point out Yunhee. Fixed it.